src/blocks/tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,105 @@
+import { type FC, useRef, useState } from 'react';
+import styled from 'styled-components';
+import * as RadixTooltip from '@radix-ui/react-tooltip';
+import type { TooltipProps } from './Tooltip.types';
+import { getTooltipPositionalCSS, getTooltipResponsiveCSS } from './Tooltip.utils';
+import { getBlocksColor } from 'blocks/Blocks.utils';
+import { tooltipCSSPropsKeys } from './Tooltip.constants';
+import { useIsVisible } from 'common';
+
+const RadixTooltipContent = styled(RadixTooltip.Content).withConfig({
+  shouldForwardProp: (prop) => !tooltipCSSPropsKeys.includes(prop as keyof TooltipProps),
+})<TooltipProps>`
+  /* Tooltip responsive styles */
+  ${(props) => getTooltipResponsiveCSS(props)}
+
+  /* Tooltip default styles */
+  display: flex;
+  flex-direction: column;
+  gap: 4px;
+  padding: 8px;
+  border-radius: 12px;
+  background-color: ${getBlocksColor('light', 'black')};
+
+  ${(props) => props.css || ''};
+`;
+
+const TooltipTitle = styled.div`
+  color: ${getBlocksColor('light', 'white')};
+  font-size: 10px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 130%;
+`;
+
+const TooltipDescription = styled.div`
```

Comment:
Let's ask zee to provide us the text components variants for title and description in tooltip. So we can use the text component directly.

If that's not possible then add support for text wrapping in the title and description.